lib/src/over_react_test/jacket.dart

```diff
@@ -100,8 +100,16 @@ class TestJacket<T extends react.Component> {
   ///
   /// > If you are rendering a function component using [mount], calling [getInstance] will throw a `StateError`.
   /// >
-  /// > * If you are trying to access an instance rendered by the function component:
-  /// >   * Use `forwardRef` to pass refs through the tree.
+  /// > * If you are trying to access an instance of some other component rendered by the function component, then try either:
+  /// >
+  /// >     i. Wrapping the component in a class-based wrapper component (such as the `Wrapper` component exported by over_react_test):
+  /// >         ```
+  /// >         final jacket = mount(Wrapper()(
+  /// >           YourFunctionComponent(),
```

Comment:
do you mean `YourFunctionComponent()()` here?

Reply:
Yes, nice catch!